### ISSUE TITLE
App Store Login

### ISSUE THREAD

**Issue by davidallenlewis** [schreiberstein/lucidagrandeelcapitan#5]
When I launch an app that was purchased in the App Store and a modal dialog comes up asking for my AppleID login... the font's are all gibberish. I uninstalled and when back to San Fran :(

**Comment by ivanzoid** [comment]
Same issue... Here is how it looks:
![screen shot 2015-11-12 at 15 35 11](https://cloud.githubusercontent.com/assets/64577/11125730/4b46fdea-898d-11e5-98a9-5a674071fc71.png)

**Comment by whitverj** [comment]
I've been seeing the same thing on Yosemite...never put 2 and 2 together that the LucidaGrandeYosemite install was the culprit.

**Comment by mdl00** [comment]
Yeah, I noticed this today as well. I suspect it's something that's not fixable (I hope I am wrong, though). Since the font displays correctly in all other dialogs I've come across, I assume that some App Store resource is specifically expecting San Francisco instead of just getting whatever the system font is.

**Comment by gerardp** [comment]
I have the same issue, any solution?

**Comment by Ry6000** [comment]
I'e had the same issues on Yosemite, but it was seemingly random. Sometimes the dialogue would be fine, and sometimes it would look like this.
![mac_app_store_sucks](https://cloud.githubusercontent.com/assets/3422003/11981472/fac33d80-a970-11e5-9c52-3b85f28b1413.png)
